### ISSUE TITLE
Feature Request - Codebox Output

### ISSUE THREAD

**Issue by pavja2** [giuspen/cherrytree#547]
It would be great if users could optionally enable codebox to record stdout as a text field within the notes document after it executes a script.

Something similar to what you get with Jupyter notebooks with a static output field below the code box could help with easily documenting the results of various scripts without copy and pasting from the terminal. It'd be especially useful for documenting the results of nmap scans and other bash one-liners.

Something like this:
![image](https://user-images.githubusercontent.com/1340709/63586118-86df6900-c598-11e9-852d-ca1e133ae786.png)

**Comment by giuspen** [comment]
Yes a good point, more probable to be delivered in the future gtkmm3 version rather than the current pygtk2
